modules/renter.go

```diff
@@ -297,3 +304,13 @@ type Renter interface {
 	// Upload uploads a file using the input parameters.
 	Upload(FileUploadParams) error
 }
+
+// RenterDownloadParameters contains all parameters that can be passed to the `/download` endpoint.
```

Comment:
yeah, it's the api that exposes the download parameters, not the download parameters that follow the api commands.